### PR TITLE
Fix NOTICE messages and result of regression test

### PR DESCRIPTION
**Pull request by thoshiai**
Currently, regression test is failed, because GROUP BY clause is inspected in advance.

### CODE REVIEW

File: createas.c [hunk 1]
```diff
@@ -1203,7 +1203,7 @@ CreateIndexOnIMMV(Query *query, Relation matviewRel, bool is_create)
 					(errmsg("could not create an index on immv \"%s\" automatically",
 							RelationGetRelationName(matviewRel)),
 					 errdetail("This target list does not have all the primary key columns, "
-							   "or this view does not contain GROUP BY or DISTINCT clause."),
+							   "or this view does not contain DISTINCT clause."),
```
Comment (yugo-n) [inline createas.c:1206]: Could you please explain why is this change required?

**Review by yugo-n — COMMENTED** [review]
Test failures were fixed in the following PR. It was my mistaken of a previous commit.
https://github.com/sraoss/pg_ivm/pull/24

So, I think we could close this PR, but before this, I would like confirm if we need to change the NOTICE message createas.c. or not. Could you please explain it?

File: createas.c (hunk 1 above)
Comment (thoshiai) [reply createas.c:1206]: I think that currently GROUP BY is not icluded in this message.  
But it needs discussion, so I will create new  pull request.
